contrib/google.golang.org/grpc.v12/grpc.go

```diff
@@ -72,8 +73,7 @@ func startServerSpanFromContext(ctx context.Context, method string, cfg *interce
 	}
 	// copy opts in case the caller reuses the slice in parallel
 	// we will add the items in extraOpts
-	optsLocal := make([]tracer.StartSpanOption, len(cfg.spanOpts), len(cfg.spanOpts)+len(extraOpts))
-	copy(optsLocal, cfg.spanOpts)
+	optsLocal := options.Copy(cfg.spanOpts...)
```

Comment:
This code already made a copy, and the new code eliminates the `extraOpts` optimization.
Not sure how much we care.

Reply:
Consistency is good. Let's keep it.

Reply:
We should probably copy this when we create the interceptor, not here. Otherwise users can modify the spanOpts at any point.

Reply:
We'll cover this in another PR.